broker/src/main/java/org/apache/rocketmq/broker/offset/ConsumerOffsetManager.java

```diff
@@ -373,6 +374,25 @@ public void setDataVersion(DataVersion dataVersion) {
         this.dataVersion = dataVersion;
     }
 
+    public boolean loadDataVersion() {
+        String fileName = null;
+        try {
+            fileName = this.configFilePath();
+            String jsonString = MixAll.file2String(fileName);
+            if (jsonString != null) {
+                ConsumerOffsetManager obj = RemotingSerializable.fromJson(jsonString, ConsumerOffsetManager.class);
+                if (obj != null) {
+                    this.dataVersion = obj.dataVersion;
+                }
+                LOG.info("load consumer offset dataVersion success, " + fileName + " " + jsonString);
```

Comment:
Log with formatter, not string concatenation: See https://medium.com/@mohanakrishnakavali/logging-efficiency-in-java-string-concatenation-vs-string-format-efa02055ddae

Reply:
There are many other places in this PR having this issue, plz fix them too.